tests/test_edf2bids.py

```diff
@@ -7,7 +7,7 @@
 
 from eye2bids.edf2bids import (
     _check_edf2asc_present,
-    _convert_edf_to_asc,
```

Comment:
tests were failing because you had not renamed this function here and it could not be imported

Reply:
AHA!!!! Thanks for explaining!

Reply:
well more precisely the test could not even start running because of that

now I am running them and figuring out why se get some failures